### DISCUSSION TITLE
How to obtain more eGRNs ?

### DISCUSSION THREAD

**Discussion by YayanFeng55** [aertslab/scenicplus#257] [Q&A]
Dear Seppe,

I was running eGRNs asscording below tutorial:

Tutorial: SCENIC+ step-by-step in the human cerebellum (https://scenicplus.readthedocs.io/en/latest/Scenicplus_step_by_step-RTD.html#). But after building eGRNs using build_grn, we only found 5 TFs and 115 genes. We have other validation approaches to this regulatory relationship. This number is quite small for me for subsequent analysis. Do you have any suggestions to generate more eGRNs ?

By the way, I have a SCENIC+ object with n_cells x n_genes = 80036 x 36438 and n_cells x n_regions = 80036 x 504458.

Thanks,
Yayan

**Comment by SeppeDeWinter** [comment]
Hi @YayanFeng55 

5 is indeed very low. Do you have an idea for how many TFs you find enriched motifs?

All the best,

Seppe

**Reply by YayanFeng55** [reply]
Hi Seppe,

I have calculated the TF number from html file. I found 1286 enriched TFs from motif enrichment analysis. 

Thanks,
Yayan

**Reply by SeppeDeWinter** [reply]
Hi @YayanFeng55 

This is unique number of TFs?

Best,

Seppe
